mapswipe_workers/mapswipe_workers/utils/validate_input.py

```diff
@@ -0,0 +1,164 @@
+import json
+import os
+
+from osgeo import ogr, osr
+
+from mapswipe_workers.definitions import (
+    DATA_PATH,
+    MAX_INPUT_GEOMETRIES,
+    CustomError,
+    logger,
+)
+
+
+def save_geojson_to_file(project_id, geometry):
+    output_file_path = (
+        f"{DATA_PATH}/input_geometries/" f"raw_input_{project_id}.geojson"
+    )
+    # check if a 'data' folder exists and create one if not
+    if not os.path.isdir("{}/input_geometries".format(DATA_PATH)):
+        os.mkdir("{}/input_geometries".format(DATA_PATH))
+
+    # write string to geom file
+    with open(output_file_path, "w") as geom_file:
+        json.dump(geometry, geom_file)
+    return output_file_path
+
+
+def validate_geometries(projectId, zoomLevel, input_file_path):
```

Comment:
This function is missing a docstring which would be helpful because of the size of the function. Also, it seems to not only validating but also changing the input and return the result. Probably it should be split into multiple functions.
Should we add an TODO comment?

Reply:
renamed and split it in the new feature branch project_type_refactoring_2 -> [8708f18](https://github.com/mapswipe/python-mapswipe-workers/pull/920/commits/8708f18c82a3b1cb3fe151e78ac647d8eac96318)